bundle/src/lib/header-bidding/prebid/prebid.spec.ts

```diff
@@ -189,7 +181,6 @@ describe('initialise', () => {
 		jest.mocked(shouldIncludeBidder).mockReturnValue(
 			jest.fn().mockReturnValue(true),
 		);
-		jest.mocked(isUserInVariant).mockReturnValue(false);
```

Comment:
I removed this as it seems like `prebid-id5` isn't in test anymore and I was using the `isUserInVariant` for grouped slots but it isn't needed anymore so I removed it from the imports.